File: gfg-Recursion/naturalNumberSum.js
const naturalNumberSum = (number) => {
    if(number == 1) return number;

    return number + naturalNumberSum(number-1);
}
let n = 5;
//5 => 5+4+3+2+1 => 15
let sum = naturalNumberSum(n)
console.log(sum)

File: gfg-Recursion/naturalNumberSum.cpp
#include<iostream>
using namespace std;

int naturalSum(int number) {
    if(number <= 1) return number;
    return number + naturalSum(number-1); 
}

int main() {
    int number = 3; //3+2+1 = 6
    //output = 6
    cout << naturalSum(number);
}